priv/riak.schema

```diff
@@ -118,11 +118,23 @@
 ]}.
 
 %% VM emulator ignore break signal (prevent ^C / ^Gq)
-{mapping, "erlang.vm.ignore_break_signal", "vm_args.+Bi", [
-  {default, "true"},
+{mapping, "erlang.vm.ignore_break_signal", "vm_args.+B", [
+  {default, true},
+  {datatype, {enum, [true, false]}},
   merge
 ]}.
 
+{translation,
+  "vm_args.+B",
+  fun(Conf) ->
+    Setting = cuttlefish:conf_get("erlang.vm.ignore_break_signal", Conf),
+    case Setting of
+      true -> "i";
+      false -> "c"
```

Comment:
I think `false` should call `cuttlefish:unset()`

Reply:
Changed as suggested, looking at `ps` output after starting riak:

Setting to `true` -

```riak -scl false -sfwi 500 -P 256000 -e 256000 -Q 262144 -A 64 -K true -W w -B i -zdbbl 32768 ...```

Setting to `false` -

```riak -scl false -sfwi 500 -P 256000 -e 256000 -Q 262144 -A 64 -K true -W w -zdbbl 32768 ...```